web-ui/src/main/resources/catalog/components/edit/onlinesrc/partials/overview-manager.html

```diff
@@ -12,7 +12,7 @@
           data-ng-repeat="thumb in relations.thumbnails">
         <img class="thumb-small"
              title="{{thumb.title | gnLocalized: lang}}"
-             data-ng-src="{{thumb.lUrl || thumb.id}}"/>
+             data-ng-src="{{onlinesrcService.getApprovedUrl(thumb.lUrl || thumb.id)}}"/>
```

Comment:
For me this code is not working, adding a breakpoint enters in the function and returns the url, but this is the html code created:

`<img class="thumb-small" title="">`


Strange stuff is that this other code in the online resources dialog works fine:

https://github.com/geonetwork/core-geonetwork/pull/5299/files#diff-bc5fb908ecf8ddd3e22bb04cd0fb066b664c25fea71c1eea53b5d0f16e5b86c6R164